coala_quickstart/generation/FileGlobs.py

```diff
@@ -24,7 +24,12 @@ def get_project_files(log_printer, printer, project_dir, non_interactive=False):
     file_globs = ["**"]
```

Comment:
Your commit contains couple of extra files which are not really required, please remove them :sweat_smile: